config/alpine/waagent.conf

```diff
@@ -69,3 +58,9 @@ OS.RootDeviceScsiTimeout=300
 
 # If "None", the system default version is used.
 OS.OpensslPath=None
+
+# Enable or disable goal state processing auto-update, default is enabled
+# AutoUpdate.Enabled=y
```

Comment:
I changed them all to be commented out, based on the issues which were opened and resulting discussion.